srcpkgs/ft2-clone/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'ft2-clone'
+pkgname=ft2-clone
+version=1.21
+revision=1
+archs="~*-musl"
```

Comment:
Can't be patched for `musl`?

Reply:
Ah, its just missing libfts, that's an easy fix

Reply:
You can drop the `archs` line if add:

```
if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
	makedepends+=" musl-fts-devel"
	configure_args+=" -DCMAKE_EXE_LINKER_FLAGS=-lfts"
fi
```

Before `do_install()`

Reply:
Done, now it should compile also for the `musl` platforms